static/sass/initial/_manifest.scss

```diff
@@ -4,4 +4,3 @@
 @import '~accoutrement-init/sass/init';
 @import 'root';
 @import 'type';
-@import 'site-message';
```

Comment:
I left the html in place, assuming there might be other reasons we want a temporary site message. I think we should either leave this in place, or remove it all if we don't think that's likely. @SondraE @jgerigmeyer want to weigh in on that?